src/components/select/bl-select.ts

```diff
@@ -241,58 +330,74 @@ export default class BlSelect extends LitElement {
       'select-wrapper': true,
       'select-open': this.opened,
       'selected': this._selectedOptions.length > 0,
-      'invalid': this._isInvalid,
+      'invalid': !this.validity.valid,
+      'dirty': this.dirty
     })}
-      tabindex="-1"
+      @keydown=${this.handleKeydown}
     >
-      ${label} ${this.inputTemplate()} ${this.menuTemplate()} ${helpMessage} ${invalidMessage}
+      ${label}
+      ${this.inputTemplate()}
+      <div class="popover" tabindex="${ifDefined(this._isPopoverOpen ? undefined : '-1')}" @bl-select-option=${this._handleSelectOptionEvent}>
+        <slot></slot>
+      </div>
+      <div class="hint">${invalidMessage} ${helpMessage}</div>
     </div> `;
   }
 
-  private get _showPlaceHolder() {
-    if (this.label && !this.labelFixed) {
-      return !this._selectedOptions.length && this._isPopoverOpen;
+  private focusedOptionIndex = -1;
+
+  private handleKeydown(event: KeyboardEvent) {
```

Comment:
looks like for the multiple select, keyboard selection is not working properly. I tried with arrow keys and then pressing Enter/Space on the options but it just toggles the popover rather than selecting the options, maybe checkbox is not getting proper focus?